dbt/adapters/greenplum/relation.py

```diff
@@ -1,32 +1,11 @@
-from dbt.adapters.base import Column
 from dataclasses import dataclass
-from dbt.adapters.base.relation import BaseRelation
-from dbt.exceptions import RuntimeException
+from dbt.adapters.postgres import PostgresRelation, PostgresColumn
 
 
 @dataclass(frozen=True, eq=False, repr=False)
-class GreenplumRelation(BaseRelation):
-    def __post_init__(self):
-        # Check for length of Greenplum table/view names.
-        # Check self.type to exclude test relation identifiers
-        if (
-            self.identifier is not None
-            and self.type is not None
-            and len(self.identifier) > self.relation_max_name_length()
-        ):
-            raise RuntimeException(
-                f"Relation name '{self.identifier}' "
-                f"is longer than {self.relation_max_name_length()} characters"
-            )
+class GreenplumRelation(PostgresRelation):
```

Comment:
same here, we need not define a new class, if the parent class is ok -- I think we can just deleted this whole file.

Reply:
Done